### DISCUSSION TITLE
Welcome to earthengine-apps Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/earthengine-apps#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajleonf** [comment]
hello giswqs, im finishing mi MC degree in meteorology and im developing an GEE app as my final thesis. Im trying to deploy my jupyter notebook app on Heroku but his free services are no longer available. So, do you know any other options to deploy the project whit free o very reduce cost? thanks a lot!

**Reply by giswqs** [reply]
See https://testdriven.io/blog/heroku-alternatives
